lib/src/api/conn.rs

```diff
@@ -5,6 +5,7 @@ use crate::api::opt::Endpoint;
 use crate::api::ExtraFeatures;
 use crate::api::Result;
 use crate::api::Surreal;
+use crate::dbs::Notification;
```

Comment:
This is actually going to be different. In the remote notifications PR I introduce a new Notification struct because it will be a different surface representation to internal representations. So we should be using the one yet-to-be-introduced. An fyi, i dont know what will merge first - probably this.

Reply:
Thanks @phughk! Let me use a separate type then.

Reply:
I have renamed `Notification::id` to `Notification::live_id` as per your PR. I also added a separate `Action` enum.